metadata.json

```diff
@@ -9,26 +9,26 @@
   "issues_url": "http://projects.theforeman.org/projects/katello/issues",
   "dependencies": [
     {
-      "name": "puppetlabs-stdlib",
+      "name": "puppetlabs/stdlib",
       "version_requirement": ">= 1.0.0"
     },
     {
-       "name": "puppetlabs-apache",
+       "name": "puppetlabs/apache",
        "version_requirement": ">= 1.0.0 < 2.0.0"
     },
     {
-       "name": "puppetlabs-mongodb"
+       "name": "puppetlabs/mongodb"
     },
     {
-       "name": "theforeman-foreman",
+       "name": "theforeman/foreman",
        "version_requirement": ">= 1.5.0"
     },
     {
-       "name": "theforeman-concat_native",
+       "name": "theforeman/concat_native",
        "version_requirement": ">= 1.3.0"
     },
     {
-       "name": "katello-qpid",
+       "name": "katello/qpid",
        "version_requirement": ">= 0.1.0"
     }
   ]
```

Comment:
Thanks, didn't realize - we tend to not use puppet module install directly.  More info here, for some reason they seem unable to match the style here as everywhere else :-\

https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#dependencies-in-metadatajson